src/components/layout/footer.js

```diff
@@ -474,6 +480,16 @@ const Footer = ({ type, is_ppc, is_ppc_redirect }) => {
                                             {localize('Payment agents')}
                                         </Link>
                                     </LinkWrapper>
+                                    <LinkWrapper>
+                                        <Link
+                                            to={deriv_developer_url}
```

Comment:
```suggestion
                                            to=""
                                            is_deriv_developer_url
```

Reply:
You'll need to pass in that prop that you created in `localized-link` for it to work.

Reply:
Done